src/ACEtk/details/verify/src/positive.hpp

```diff
@@ -0,0 +1,27 @@
+template< typename Range >
+inline bool positive( Range& array ){
```

Comment:
Do you mean lvalue ref?

Reply:
Seems this asked to be misused. If the range is sorted, this does a lot of redunant work. How about we design the api to apply to a scalar

Reply:
predicate functions begin with `is` by convention, e.g. `isPositive`

Reply:
It's true if the range is sorted, this is not the most efficient method. However, there will be some arrays/ranges that must be checked for positivity that may or may not be sorted. There is a method for checking positivity on sorted ranges.

Reply:
I'm not sure what you mean by:
 > How about we design the api to apply to a scalar